feature/platform/fabric/otg_tests/sampled_backplane_capacity_counters_test/README.md

```diff
@@ -0,0 +1,79 @@
+# gNMI-1.19: gNMI subscribe with sample mode for backplane capacity counters
+
+## Summary
+WBB is required to support gNMI Subscribe with sample mode for various counters.
+This test if to verify that DUT supports gNMI Subscribe with sample mode, updating
+the available backplane capacity counters correctly while forwarding traffic
+
+*   Get backplace capacity before any traffic is sent
+
+*   Send some traffic and wait for the sample to be collected
+
+*   Send more traffic and wait for the sample to be collected
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_9.testbed
+
+## Procedure
+
+*   Connect DUT port-1 through 9 to ATE port-1 through 9
+
+*   Configure IPv4/IPv6 addresses on all the 9 interfaces
+
+*   Using gNMI subscribe with sample mode and an interval of 10 seconds
+    validate the backplane capacity before any traffic is sent and during the phases
+    This will happen at 0 Seconds 
+
+*   Phase 1 (initiate traffic immediately):
+
+    *   Initiate traffic as per below threshold:
+    
+         Port number   | Interface1(line rate %)
+        -------------- | -----------------------
+        Port1          | 20
+        Port2          | 20
+        Port3          | 20
+        Port4          | 20
+        Port5          | 20
+        Port6          | 20
+        Port7          | 20
+        Port8          | 20
+        Port9          | 20
+
+*   We should get second sample at 10 seconds
+
+*   Phase 2 (after 10 seconds):
+
+    *   Initiate traffic as per below threshold:
+    
+         Port number   | Interface1(line rate %)
+        -------------- | -----------------------
+        Port1          | 80
+        Port2          | 80
+        Port3          | 80
+        Port4          | 80
+        Port5          | 80
+        Port6          | 80
+        Port7          | 80
+        Port8          | 80
+        Port9          | 80
+
+*   We should get third sample at 10 seconds
+
+*   Ensure available backplane capacity shows a decrease at every sample interval
```

Comment:
How you can be sure that DUT follow subscribe interval and not run it at "target_defined" that are by coincidence also 10sec?

Perhaps 2ns run with 20 sec interval will provide this certanity.

Reply:
Specify "SAMPLE" as the "SubscriptionMode" and not "TARGET_DEFINED"
like done here: https://github.com/openconfig/featureprofiles/blob/3eb2a212fc4a88f66cd50c130466d448931822cc/feature/gnmi/otg_tests/telemetry_interface_packet_counters_test/telemetry_interface_packet_counters_test.go#L183

Reply:
+1 to @rohit-rp comment.  This test must explicitly use a gnmi subscription in `SAMPLE` mode.